examples/r2i/tensorrt/stb_image_resize.h

```diff
@@ -0,0 +1,2746 @@
+/* stb_image_resize - v0.94 - public domain image resizing
```

Comment:
This file is duplicated in tensorflow, tflite and tensorrt. We should refactor it!